dev_tools/scripts/install-sda-deps.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -e
+
+# Workaround for some MacOS installations
+#export PATH=$PATH:/home/ubuntu/.local/bin
+
+if [[ ! -d sda-helm ]]; then
+  git clone https://github.com/neicnordic/sda-helm
+fi
```

Comment:
Why are you cloning the helm repo?
This test is run in that repo so cloning it means you get whats in master and not the branch that needs to be tested.